README-detailed.md

```
@@ -0,0 +1,922 @@
+
+5. **Be Responsive**: Be prepared to address feedback and make requested changes.
+
+### Questions?
+
+If you have any questions about contributing, feel free to [open an issue](https://github.com/jamcalli/pulsarr/issues) with the label "question".
+
+## Contributors
+
+<!-- readme: contributors -start -->
+<table>
+	<tbody>
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)